From c010a95d8c9aeda827d78884f93f152706ea6232 Mon Sep 17 00:00:00 2001
From: Philippe Normand <philn@igalia.com>
Date: Fri, 3 Apr 2020 10:52:29 +0100
Subject: [PATCH] meson: Bump to latest version of the 3.24 branch

3.24.16 was tagged without bumping the version in the Meson build definitions.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 0c31ca0f30..a7acd96ce7 100644
--- a/meson.build
+++ b/meson.build
@@ -1,5 +1,5 @@
 project('gtk+-3.0', 'c',
-        version: '3.24.14',
+        version: '3.24.16',
         default_options: [
           'buildtype=debugoptimized',
           'warning_level=1'
-- 
2.25.1

